addons/account/models/account_move.py

```diff
@@ -2918,6 +2918,37 @@ def is_internal_partner(partner):
         move.message_subscribe(list(all_followers_ids))
         return move
 
+    def _post_check_non_negative(self):
```

Comment:
since this only applies on invoices, perhaps we can rename the method to indicate such? or we can move the `if` back to the `_post` method

Reply:
I dont feel to move the invoice check back to the post (outside from this separated method) to be fully adjustable by any localization module.
So just renamed the method to make it clear it checks the invoices as you requested.